bootstrap/src/index.ts

```diff
@@ -72,14 +79,14 @@ const withAsync = (execute: ExecuteWrappedResponse | ExecuteSync): ExecuteWrappe
 }
 
 // Execution helper async => sync
-const executeSync = (execute: ExecuteWrappedResponse): ExecuteSync => {
+const executeSync = (execute: ExecuteWrappedResponse, config?: Config): ExecuteSync => {
   // TODO: Try to init middleware only once
   // const initMiddleware = withMiddleware(execute)
 
   // Return sync function
   return (data: AdapterRequest, callback: any) => {
     // We init on every call because of cache connection broken state issue
-    return withMiddleware(execute)
+    return withMiddleware(execute, config)
```

Comment:
Just send the function to the middlewares. If some needs `execute.config` they can read it, if not they just `execute.call`. 

```suggestion
    return withMiddleware(execute)
```